doc/Extensions/Applications.md

```diff
@@ -1172,6 +1172,45 @@ sees them as being.
 For questions about what the various values are/mean, please see the
 nvidia-smi man file under the section covering dmon.
 
+## Opensearch\Elasticsearch
+
+### SNMP Extend
+
+1. Download the script onto the desired host.
+```
+wget https://raw.githubusercontent.com/librenms/librenms-agent/master/agent-local/rocks.sh -O /etc/snmp/opensearch
```

Comment:
wrong url

Reply:
Derp! Thanks, dude!

Copied it from the other another line and forgot to change that bit. =^.^=